include/feeding/util.hpp

```diff
@@ -37,8 +37,9 @@ static const std::vector<std::string> FOOD_NAMES = {"apple",
                                                     "spinach",
                                                     "kale"};
 
+// TODO(egordon): Reference from YAML file
```

Comment:
Hmm, isn't FOOD_NAMES in the yaml, and can't we add BEST_ACTIONS to the yaml? That might be what this TODO was referring to.

Reply:
I believe Ethan addressed this